spec/features/providers/ucas_alerts_spec.rb

```diff
@@ -0,0 +1,82 @@
+require "rails_helper"
+
+feature "Edit UCAS email alerts", type: :feature do
+  let(:page) { PageObjects::Page::Organisations::UcasContactsAlerts.new }
+  let(:org_ucas_contacts_page) { PageObjects::Page::Organisations::UcasContacts.new }
+  let(:provider) { build(:provider, send_application_alerts: nil) }
+  let(:current_recruitment_cycle) { build :recruitment_cycle }
+
+  before do
+    stub_omniauth
+    stub_api_v2_resource(provider)
+    visit alerts_provider_ucas_contacts_path(provider.provider_code)
+  end
+
+  scenario "selecting and saving an option" do
+    expect(current_path).to eq alerts_provider_ucas_contacts_path(provider.provider_code)
+    expect(page.title).to have_content("Email alerts for new applications")
+    expect(page.main_heading).to have_content("Email alerts for new applications")
+    expect(page).to have_alerts_enabled_fields
+    expect(page.alerts_enabled_fields).to have_all
```

Comment:
do you actually need both to see if its there and to see if its checked?

i think the below will be enough
```
    expect(page.alerts_enabled_fields.all).not_to be_checked
    expect(page.alerts_enabled_fields.none).not_to be_checked
```

if it can see it then it can't be checked or not

Reply:
good point. would have made errors slightly clearer perhaps but not worth it for extra code. done. cheers.